src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceModelHttpMessageHandler.NetNative.cs

```diff
@@ -79,11 +79,6 @@ public bool SupportsCertificateValidationCallback
         {
             get { return false; }
         }
-
-        protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
-        {
-            return base.SendAsync(request, cancellationToken);
-        }
```

Comment:
Question only: I can see why this SendAsync is being removed, because it was only calling the base.  But in the non-NET Native version of this class, you are adding a new override of SendAsync to await the WriteCompletedTask.  Could you explain in this PR thread why the NET Native version is not doing the same override?